getting_started/first_2d_game/03.coding_the_player.rst

```diff
@@ -0,0 +1,393 @@
+.. _doc_your_first_2d_game_coding_the_player:
+
+Coding the player
+=================
+
+In this lesson, we'll add player movement, animation, and set it up to detect
+collisions.
+
+To do so, we need to add some functionality that we can't get from a built-in
+node, so we'll add a script. Click the ``Player`` node and click the "Attach
+Script" button:
+
+.. image:: img/add_script_button.png
+
+In the script settings window, you can leave the default settings alone. Just
+click "Create":
+
+.. note:: If you're creating a C# script or other languages, select the language
+          from the `language` drop down menu before hitting create.
+
+.. image:: img/attach_node_window.png
+
+.. note:: If this is your first time encountering GDScript, please read
+          :ref:`doc_scripting` before continuing.
+
+Start by declaring the member variables this object will need:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    extends Area2D
+
+    export var speed = 400 # How fast the player will move (pixels/sec).
+    var screen_size # Size of the game window.
+
+ .. code-tab:: csharp
+
+    using Godot;
+    using System;
+    
+    public class Player : Area2D
+    {
+        [Export]
+        public int speed = 400; // How fast the player will move (pixels/sec).
```

Comment:
According to the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html#naming-conventions), this should be capitalised (same for `screenSize` below).

Reply:
Please feel free to edit the csharp code in a separate pull request. I won't touch the existing csharp code, I'm just splitting the existing one-page tutorial

Reply:
Noted, and will do 👌

Reply:
Thanks, I'll rebase and merge now so you can work on top of these changes

Reply:
Oops!
I didn't see your conversation and made a PR concerning these changes [here](https://github.com/godotengine/godot-docs/pull/4723).
@paulloz, I hope you haven't started yet

Reply:
I was packaging that in a bigger PR. But that's no big deal, don't worry.